### ISSUE TITLE
no-unused broken in TypeScript 3: TypeError: ts.unescapeIdentifier is not a function

### ISSUE THREAD

**Issue by AlexanderOMara** [ajafff/tslint-consistent-codestyle#90]
`no-unused` appears to be broken when using TypeScript 3:

This code:

```ts
const TEST = 1;
```

Produces this error:

> The 'no-unused' rule threw an error in '/path/to/file.ts':
> TypeError: ts.unescapeIdentifier is not a function

**Comment by ajafff** [comment]
This is fixed upstream in `tsutils`. The fix is available in the versions 2.x and 3.x of the package. So you probably just need to update the version of that dependency in your lockfile.

**Comment by ahanriat** [comment]
I'm having the same issue, I'm not very keen to change my lockfile manually though

**Comment by ajafff** [comment]
@ahanriat If you don't update your lockfile regularly, you will have lot's of outdated dependencies. You will miss all the important bugfixes like the one that shipped in `tsutils`.

Anyways, this issue was actually fixed before it was reported. I'm not going to rush an update of `tsutils` as this is a breaking change.
